test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -21,6 +21,24 @@
 KUBE_ROOT="/home/kubernetes"
 KUBE_BINDIR="${KUBE_ROOT}/kubernetes/server/bin"
 
+function config-ip-firewall {
+  echo "Configuring IP firewall rules"
+  # The GCI image has host firewall which drop most inbound/forwarded packets.
```

Comment:
I don't know off the top of my head but it looks like this hunk is straight out of `cluster/gce/gci/configure-helper.sh`. @freehan @andyzheng0831 `git blame` points to you two, any background info on this?